### DISCUSSION TITLE
is there a way to make lost-pixel to render pages with the same engine on different OS

### DISCUSSION THREAD

**Discussion by studioab** [lost-pixel/lost-pixel#257] [Q&A]
Here is our scenario:
 - Build agent runs lost-pixel on Ubuntu
 - Dev 1 runs it locally on MacOs
 - Dev 2 runs it locally on PC

lost-pixel fails on different platforms because results are slightly different because of the available system fonts and rendering engines

what is the suggested workaround?

**Comment by d-ivashchuk** [comment]
Hey, we are working on the solution to run lost pixel in docker container to mitigate the OS rendering specifics. If you'd like to contribute and help us get there - ping me, I'd gladly accept some help 🙏🏼

**Comment by d-ivashchuk** [comment]
we have shipped the docker version of lost pixel run :D you can beta test it by `npx lost-pixel docker` and see if the resulting test runs will be the same for the devs on different platforms!
